case_studies/strong_lensing/run_sim.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+# Script to automate running the galsim command with the yaml configuration
+
+# Default Values
+num_files=3
+num_galaxies=3
+img_size=500
+
+
```

Comment:
If I run the script without arguments, it just prints a message and exits cleanly:
```
$ ./run_sim.sh 
Starting data generation...
$
```

Would you change it to run with sensible default if the user provides no arguments? (And perhaps to print a message about what defaults it is using.)

Reply:
Ah, I see the output was getting written to a log file. Perhaps send the output to stdout too (or instead), because otherwise a user won't no were to look.